rdbms/src/test/java/org/spine3/server/storage/jdbc/util/SerializerShould.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2016, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.storage.jdbc.util;
+
+import org.junit.Test;
+import org.spine3.test.project.ProjectId;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.spine3.testdata.TestAggregateIdFactory.newProjectId;
+
+/**
+ * @author Alexander Litus
+ */
+@SuppressWarnings("InstanceMethodNamingConvention")
+public class SerializerShould {
+
+    @Test
+    public void serialize_and_deserialize_message() {
+        final ProjectId expected = newProjectId();
+
+        final byte[] bytes = Serializer.serialize(expected);
+        assertTrue(bytes.length > 0);
+
+        final ProjectId actual = Serializer.deserialize(bytes, expected.getDescriptorForType());
+        assertEquals(expected, actual);
+    }
+}
```

Comment:
@AlexanderLitus 

Two general points:

- Extensibility for queries (allow to change queries for framework users).
- Make Hikari CP usages configurable.

Both of those should be documented. E.g. to customize a query, do a), b) and c).